src/config.ts

```diff
@@ -278,7 +278,7 @@ export function parseInput(input: string, config: Config, forbidden: Forbidden[]
 
   // remove forbidden words
   const positive = input.split(/,\s*/g).filter((word) => {
-    word = word.replace(/[^a-z0-9]+/g, ' ').trim()
+    word = word.replace(/[\x00-\x7f]/g, (s) => s.replace(/[^0-9a-zA-Z]/, ' ')).trim() // ascii except /a-zA-Z0-9/
```

Comment:
```suggestion
    word = word.replace(/[\x00-\x7f]/g, s => s.replace(/[^0-9a-zA-Z]/, ' ')).replace(/\s+/, ' ').trim()
```

should also handle consecutive spaces

Reply:
didn't know the neccessariness, but ok